src/controllers/customercontroller.js

```diff
@@ -0,0 +1,70 @@
+class CustomerController extends Controller
+{
+    view()
+    {
+        this._customerHelper();
+    }
+
+    generateOrder()
+    {
+        var products = this.constructor._makeOrder();
+
+        var orderSize = products.reduce((sum, prod) => sum + prod.shelfSize(), 0);
+        var orderCost = products.reduce((sum, prod) => sum + prod.value(), 0);
+
+        var customer = {
+            name: 'Henk', // Find way to generate names
+            products: products
+        };
+
+        MODEL.customers.push(customer);
+
+        this._updateOrderView(customer);
+
+        toastr.success("New customer is waiting!");
+    }
+
+    static registerEvent()
+    {
+        //
+    }
+
+    _updateOrderView(customer)
+    {
+        $.get(
+            "src/views/template/customerorder.html",
+            function (progressBarView) {
```

Comment:
Might want to change the name here from progressBarView into something more appropriate :)